common/utils/src/main/resources/error/error-conditions.json

```
@@ -3135,6 +3135,12 @@
     },
     "sqlState" : "42K0E"
   },
+  "INVALID_TIMEZONE": {
+    "message": [
+      "ZoneOffset represents the time difference from UTC and must be in the range from -18:00 to +18:00."
```

Unless the input is a literal the user will not know what the invalid offset is. You need to provide the value. Also teh advise is incomplete
```suggestion
      "The timezone: <timeZone> is invalid.  The timezone must be either a region-based zone ID or a zone offset. Region IDs must have the form ‘area/city’, such as ‘America/Los_Angeles’. Zone offsets must be in the format ‘(+|-)HH’, ‘(+|-)HH:mm’ or ‘(+|-)HH:mm:ss’, e.g ‘-08’, ‘+01:00’ or ‘-13:33:33’., and must be in the range from -18:00 to +18:00. 'Z' and 'UTC' are accepted as synonyms for '+00:00'."
```

---

Looks way better. Thanks!